upstream-community-operators/federation/federation.v0.0.7.clusterserviceversion.yaml

```diff
@@ -3,14 +3,14 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
-  name: federation.v0.0.6
+  name: federation.v0.0.7
   namespace: placeholder
   annotations:
```

Comment:
Should I add `metadata.annotations.alm-examples` as a separate PR?

Reply:
Yes please